semgrep_output_v1.atd

```diff
@@ -836,8 +836,7 @@ type finding <ocaml attr="deriving show"> = {
   end_line: int;
   end_column: int;
   message: string;
-  (* TODO: use variant *)
-  severity: int;
+  severity: abstract; (* int|string until minimum version exceeds 1.32.0, then string *)
```

Comment:
What are valid values for the int and for the string, and how to convert one from the other? Is it documented somewhere else?